docs/src/content/en/docs/getting-started/installation.mdx

```diff
@@ -76,14 +80,14 @@ Start with a pre-built template that demonstrates specific use cases:
 npx create-mastra@latest --template template-name
 ```
 
+> Browse available templates and learn more in [Templates](/docs/getting-started/templates).
```

Comment:
Two quotes directly underneath each other looked weird so I moved this above. Also makes sense because the code block above talks about the generic template option while below a specific example is given.